tests/spread/general/virtual-packages/task.yaml

```diff
@@ -0,0 +1,32 @@
+summary: Build a snap that has virtual stage and build packages
```

Comment:
this should probably be enabled for 20.04 as well.

Reply:
Prior to enabling 20.04 suite for all of general?

Reply:
Done